src/compile/data.ts

```diff
@@ -650,6 +762,11 @@ export namespace stackScale {
     return null;
   }
 
+  export function parseLayer(model: LayerModel) {
+    // TODO
```

Comment:
Do we have an issue for this?  If not, please file one in the follow-up milestone